client/test/contact_store.spec.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+const assert = require('chai').assert;
+
+const mockeryUtils = require('./utils/mockery_utils');
```

Comment:
@frankrousseau: I still not find this file; maybe you have forgotten to re-add it when you rebase your commits?